Cargo.toml

```diff
@@ -77,7 +77,7 @@ version = "1.5.3"
 version = "0.7.4"
 
 [dependencies.snarkvm-curves]
-version = "0.7.5"
+version = "0.7.6"
 default-features = false
 
 [dependencies.snarkvm-gadgets]
```

Comment:
Is there a reason why some of the snarkvm-? are still 0.7.5 rather than 0.7.6? Like the snarkvm-gadgets here.

Reply:
Nope. There's no reason. We even had 0.7.3 somewhere. Just pushed the updates for everything.

Reply:
Got it. Can confirm that this issue has been solved -> I checked the output of `cargo tree | grep snarkvm` and all of them are 0.7.6 now.